### PR TITLE
⏩ Migrate External Assets Page

### PR DESCRIPTION
**Pull request by MeganSteenkamp**
Story: https://www.notion.so/centrapay/Migrate-API-Documentation-c6d64de29a2245358a5e9b7eed5edbcc?pvs=4

Test plan: Expect to see migrated draft page with content matching legacy page.

### CODE REVIEW

**Review by AndyClifford — APPROVED** [review]

File: src/content/api/external-assets.mdx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      "balanceUpdatedAt": "2020-06-10T15:30:00.000Z",
+      "expiresAt": "2020-12-31T00:00:00.000Z",
+      "createdAt": "2020-05-01T12:30:00.000Z"
+    }
+    ```
+  </CodePanel>
+
+  ### Errors
+  <Properties>
+    <Property name="403" type="UNKNOWN_ASSET" error>
```
[the remark this answers is not in the document]
Comment (MeganSteenkamp) [reply src/content/api/external-assets.mdx:96]: Ty @AndyClifford. Have we got WIP to migrate the existing ones? I can see assets and bank account approvals use the old way which is why I got confused

[the remark this answers is not in the document]
Comment (AndyClifford) [reply src/content/api/external-assets.mdx]: I did all of them but looks like some of them have been overwritten

[the remark this answers is not in the document]
Comment (AndyClifford) [reply src/content/api/external-assets.mdx]: Are you able to include those in this PR?

[the remark this answers is not in the document]
Comment (MeganSteenkamp) [reply src/content/api/external-assets.mdx]: @AndyClifford I've addressed in Pull Request #829

**Pull request merged**